### PR TITLE
go/staking: Read token information from state when available

### PR DESCRIPTION
**Pull request by kostko**
This unifies handling of the two remaining attributes so everything is consistently read from state, in a backwards compatible manner.

### CODE REVIEW

File: go/staking/api/api.go [hunk 1]
```diff
@@ -1183,6 +1183,13 @@ type Genesis struct {
 
 // ConsensusParameters are the staking consensus parameters.
 type ConsensusParameters struct { // nolint: maligned
+	// TokenSymbol is the token's ticker symbol.
+	// Only upper case A-Z characters are allowed.
```
Comment (peternose) [inline go/staking/api/api.go:1187]: Where is this enforced?

File: go/staking/api/api.go [hunk 2]
```diff
@@ -1183,6 +1183,13 @@ type Genesis struct {
...
+	TokenSymbol string `json:"token_symbol,omitempty"`
```
Comment (peternose) [inline go/staking/api/api.go:1188]: When/where will these two fields be set?

**Review by peternose — APPROVED** [review]

File: go/staking/api/api.go (hunk 1 above)
Comment (kostko) [reply go/staking/api/api.go:1187]: It is not, but it cannot really be changed outside an upgrade handler.

File: go/staking/api/api.go (hunk 2 above)
Comment (kostko) [reply go/staking/api/api.go:1188]: Currently only via an upgrade handler. I would keep it this way for now.

**Pull request merged**